### DISCUSSION TITLE
Is check existence asked a bit too often for benches and similar? (changed)

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#2381] [Q&A] [answered]
And least in my case benches are not disappearing so often, and so far I just keep answering "yes, it is here".

Is it just me? I thought about changing this 4 to 5 (presumably years).

-----------------------------

PS: relevant code is 

```
        ) or (
          (
            amenity = clock
            or amenity = bench
            or amenity = post_box
            or leisure = picnic_table
            or leisure = firepit
            or amenity = vending_machine and vending = public_transport_tickets
            or tourism = information and information ~ board|terminal|map
            or advertising ~ column|board|poster_box
          )
          and (${lastChecked(4.0)})
        )) and access !~ no|private
```

**Comment by HolgerJeromin** [comment]
ref https://github.com/streetcomplete/StreetComplete/pull/2335#issuecomment-740453460

**Comment by FloEdelmann** [comment, marked as answer]
ref https://github.com/streetcomplete/StreetComplete/commit/3a15d9a28aa472e6cefca0ac95c4fac3419cdeea (ask for waste baskets only every 4 years)

**Reply by matkoniecz** [reply]
See also https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/existence/CheckExistence.kt#L52 - nowadays both benches and waste baskets are asked every 6 years

**Comment by timothywashere** [comment]
Maybe even depending on material? A metal bench in an urban area is more likely to be around in 5 years than a timber bench in a coastal location?

**Reply by mnalis** [reply]
Maybe... but given that `material=*` on `amenity=bench` was not deemed interesting enough for SC quest itself (https://github.com/streetcomplete/StreetComplete/issues/3132), probably not...
